CHANGELOG.md

```diff
@@ -192,10 +192,22 @@ Released 2021-10-09
 - Moves All JWT validation to decoding function from resource
 
 
+### Version 0.5.1
+
+- Extends CORS capabilities and adds relevant unit tests.
 
 ### Version 0.5.2
 Released 2022-04-12
 
 - Adds Event Broker to support Asynchronous event handling with AWS Lambda
 - Introduces isort for better sorting and related checks
-- Updates dependencies
\ No newline at end of file
+- Updates dependencies
+
+
+### Version 0.5.3
+Released 2022-04-15
+
+- Adds EventApi and EventAware Resource
+- Adds Response as a part of Resource, so it can be easily accessed by post hook.
+- Improves mypy situation for tests
+- Makes Resource's post_request_hook error proneŻ
```

Comment:
Typo: "_prone**Ż**_" but rather you wanted to write "_error-proof_" or "_fault-tolerant_"